packages/core/src/classes.ts

```diff
@@ -279,6 +279,7 @@ export interface Space extends Doc {
  */
 export interface Account extends Doc {
   email: string
+  owner: boolean
```

Comment:
Let's use role instead to minimize further migrations.
```suggestion
  role: 'owner' | 'admin'
```